src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -198,23 +162,8 @@ void VM_Version::initialize() {
   // Create and print feature-string.
   char buf[(num_features+1) * 16]; // Max 16 chars per feature.
   jio_snprintf(buf, sizeof(buf),
-               "ppc64%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s",
-               (has_fsqrt()   ? " fsqrt"   : ""),
-               (has_isel()    ? " isel"    : ""),
-               (has_lxarxeh() ? " lxarxeh" : ""),
-               (has_cmpb()    ? " cmpb"    : ""),
-               (has_popcntb() ? " popcntb" : ""),
-               (has_popcntw() ? " popcntw" : ""),
-               (has_fcfids()  ? " fcfids"  : ""),
-               (has_vand()    ? " vand"    : ""),
-               (has_lqarx()   ? " lqarx"   : ""),
-               (has_vcipher() ? " aes"     : ""),
```

Comment:
" aes" seems to be still needed in the feature string. See TestAESIntrinsicsOnSupportedConfig.java and TestAESIntrinsicsOnUnsupportedConfig.java.